<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Processing</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="loadImageButton">
      <property name="text">
       <string>Load Image</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="imageLabel">
      <property name="text">
       <string>Image will appear here</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="morphologyButton">
      <property name="text">
       <string>Morphological Processing</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="smoothingButton">
      <property name="text">
       <string>Smoothing Filter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
